handlers/devicetype.go

```diff
@@ -21,6 +21,21 @@ func NewDeviceTypeHandler(service ports.DeviceTypeService) *DeviceTypeHandler {
 	}
 }
 
+// Handle API endpoint for listing all device types.
```

Comment:
I see you created a function to be able to list all device types, but you don't expose an endpoint to reach this.

Reply:
That was an issue on our side, it has been removed.